src/3d/qgsabstract3dengine.h

```diff
@@ -66,6 +67,8 @@ class _3D_EXPORT QgsAbstract3DEngine : public QObject
      */
     QgsAbstract3DEngine( QObject *parent = nullptr );
 
+    QElapsedTimer mTimer;
```

Comment:
Is this for debugging? Probably worth removing, or at least it should not be a public member variable...